### PR TITLE
datetime to astropy Time: Database

### PR DESCRIPTION
**Pull request by vn-ki**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -47,16 +47,20 @@ jobs:
...
+
       - checkout
       - run: *apt-install
       - run: *docs-install
       # Copy and paste this line for every submodule for which the tests pass
       - run: conda install openjpeg
       - run: python -m "sunpy.data.sample"
       - run: python setup.py test -P time
-      - run: python setup.py test -P net --online
+      # - run: python setup.py test -P net --online
```
Comment (Cadair) [inline .circleci/config.yml]: I assume this was because VSO was timing out? It should be back now.

File: sunpy/database/tables.py [hunk 2]
```diff
@@ -694,15 +689,17 @@ def entries_from_file(file, default_waveunit=None,
...
             # FITS standard, but many FITS files use it in their header
             elif key in ('DATE-END', 'DATE_END'):
-                entry.observation_time_end = parse_time(
-                    value,
-                    _time_string_parse_format=time_string_parse_format
-                )
+                try:
+                    dt = parse_time(value).datetime
+                except ValueError:
+                    dt = Time.strptime(value, time_string_parse_format).datetime
```
Comment (Cadair) [inline sunpy/database/tables.py]: What happens if you get here and `time_string_parse_format=''`?

Comment (vn-ki) [reply sunpy/database/tables.py]: `ValueError`

File: .circleci/config.yml (hunk 1 above)
Comment (vn-ki) [reply .circleci/config.yml]: That's because net takes almost half an hour to complete running.

File: sunpy/database/tables.py (hunk 2 above)
Comment (Cadair) [reply sunpy/database/tables.py]: :+1:

File: .circleci/config.yml (hunk 1 above)
Comment (Cadair) [reply .circleci/config.yml]: Why don't we put the online tests into a seperate `astropy-time-online` circle ci job?

Comment (vn-ki) [reply .circleci/config.yml]: Yeah. Sounds good.

Comment (Cadair) [reply .circleci/config.yml]: Do you want to add that here and then we can merge this?

Comment (vn-ki) [reply .circleci/config.yml]: Yeah. I'll add it now.

File: .circleci/config.yml [hunk 3]
```diff
@@ -55,12 +55,27 @@ jobs:
       - run: *docs-install
       # Copy and paste this line for every submodule for which the tests pass
       - run: conda install openjpeg
-      - run: python -m "sunpy.data.sample"
       - run: python setup.py test -P time
-      # - run: python setup.py test -P net --online
       - run: python setup.py test -P physics
       - run: python setup.py test -P coordinates
+
+  astropy-time-online:
```
Comment (Cadair) [inline .circleci/config.yml]: You need to add this job to the workflows section at the end for it to get executed.

Comment (vn-ki) [reply .circleci/config.yml]: Yeah. Forgot.

**Pull request merged**